src/lib/components/AgGrid.react.js

```diff
@@ -770,6 +772,4 @@ const _getAsync =
         );
     };
 export const getApi = _get('getApi');
-export const getColumnApi = _get('getColumnApi');
```

Comment:
So `getColumnApi` functionality was all merged into `getApi` (and similar for async)? Given that we're exporting these, should we mention this in the [migration guide](https://github.com/plotly/ddk-dash-docs/pull/1938/files#diff-07b135ac5c223e868fdf5b83660e76d692f891059899a28d89b739e9426c7916)?

Reply:
We could mention it in the migration guide, but we haven't yet documented how to use these functions, so it might be a little confusing.

Reply:
Updated [Migration Guide](https://github.com/plotly/ddk-dash-docs/pull/1938/commits/08d0c67537fe3117b57e7b35235e1e751ff4bfe6)  Also updated the notes to add docs for `getApi`